### DISCUSSION TITLE
CPPC Option

### DISCUSSION THREAD

**Discussion by enesuzun2002** [DavidS95/Smokeless_UMAF#30] [Q&A] [answered]
So i have an Acer Nitro 5 AN515-44 with ryzen 7 4800h i want to enable CPPC to use new amd_pstate driver on linux but i don't want to brick my device any have ever tried this option is it safe to use?

**Comment by aviallon** [comment, marked as answer]
@enesuzun2002 I did softbrick my Steam Deck by trying to activate it, so your mileage may vary.
I easily recovered by resetting the CMOS though.

**Reply by enesuzun2002** [reply]
thanks for the response

**Comment by sammcj** [comment]
I was looking for CPPC for the same reason - to get the new amd_pstate driver working properly in linux so that Pstates beyond 1 are available.

Is there a way I can try it out knowing that I'd have to reset my CMOS if required @enesuzun2002 ?

**Reply by aviallon** [reply]
Yes, you can, using Smokeless_UMAF, there is a CPPC option hidden in some of the menus.
You'll have to dig it up though, I can't remember where I found it.

**Comment by mateushonorato** [comment]
I have the model as you, Acer Nitro AN515-44, with a Ryzen 5 4600H. I just managed to enable CPPC and use the amd_pstate driver on Linux successfully using this tool.

**Reply by enesuzun2002** [reply]
I have enabled it too, but in my case it's drawbacks were more significant then the benefits

**Reply by LFRon** [reply]
> I have the model as you, Acer Nitro AN515-44, with a Ryzen 5 4600H. I just managed to enable CPPC and use the amd_pstate driver on Linux successfully using this tool.

Where you find the CPPC option?

**Comment by LFRon** [comment]
I'm using Ryzen 7 8845HS,but I cannot find CPPC option at the SMU command line options

**Comment by mateushonorato** [comment]
The CPPC option can be found under `Device Manager > AMD CBS > NBIO Common Options > SMU Common Options > CPPC`
For me I just set it to enabled and left all other values as default. It didn't really added performance, but battery backup was slightly improved when using powersave mode.

**Reply by LFRon** [reply]
> The CPPC option can be found under `Device Manager > AMD CBS > NBIO Common Options > SMU Common Options > CPPC` For me I just set it to enabled and left all other values as default. It didn't really added performance, but battery backup was slightly improved when using powersave mode.

But my Ryzen 7 8845HS Laptop SMU option is at "Device Manager > AMD CBS > SMU Common Options" and which doesn't have CPPC option😭

**Reply by Naitrate** [reply]
> > The CPPC option can be found under `Device Manager > AMD CBS > NBIO Common Options > SMU Common Options > CPPC` For me I just set it to enabled and left all other values as default. It didn't really added performance, but battery backup was slightly improved when using powersave mode.
> 
> But my Ryzen 7 8845HS Laptop SMU option is at "Device Manager > AMD CBS > SMU Common Options" and which doesn't have CPPC option😭

Same. I can't seem to find it on my Alienware M17 R5.

**Reply by mateushonorato** [reply]
Which version did u guys used? I used the main version, if you're using the beta maybe that's the reason.

**Reply by aviallon** [reply]
You also need to set `amd_pstate=passive` (or any other valid value) for it to have any effect.

**Reply by shaogme** [reply]
I use 8845hs and have the same problem that smu is missing. No matter beta or main, it is still missing.
